### PR TITLE
Marketplace - Skipped zipping of old format reputation files

### PR DESCRIPTION
**Pull request by IkaDemisto**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Server version 6.0.0 will not support old format reputation json files. Skipped it while zipping and additionally removed zipping of not necessary files that were used for debugging (pack_metadata.json). 

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

### CODE REVIEW

File: Tests/store_packs.py [hunk 1]
```diff
@@ -236,6 +239,17 @@ def zip_pack(self):
...
+                        # skipping unzipping of unwanted files
+                        if f.startswith('.') or f in [Pack.AUTHOR_IMAGE_NAME, Pack.USER_METADATA]:
+                            print_warning(f"Skipping zipping {f} for {self._pack_name} pack")
+                            continue
+
+                        current_directory = root.split(os.path.sep)[-1]
+
+                        if current_directory == 'Misc' and not fnmatch.fnmatch(f, 'reputation-*.json'):
+                            # reputation in old format aren't supported in 6.0.0 server version
+                            print_warning(f"Skipped zipping {f} for {self._pack_name} pack")
```
Comment (yuvalbenshalom) [inline Tests/store_packs.py]: @IkaDemisto soul dent you add `continue` here?

Comment (IkaDemisto) [reply Tests/store_packs.py]: @yuvalbenshalom correct

File: Tests/store_packs.py [hunk 2]
```diff
@@ -236,6 +239,18 @@ def zip_pack(self):
                     dirs[:] = [d for d in dirs if d not in Pack.EXCLUDE_DIRECTORIES]
 
                     for f in files:
+                        # skipping unzipping of unwanted files
```
Comment (yuvalbenshalom) [inline Tests/store_packs.py]:
```suggestion
                        # skipping zipping of unwanted files
```

Comment (IkaDemisto) [reply Tests/store_packs.py]: fixed

**Pull request merged**